content/en/users/tutorials/htc-job-submission/_index.md

```diff
@@ -0,0 +1,397 @@
+ $ condor_transfer_data -name condorce1.ciemat.es -pool condorce1.ciemat.es:9619 97412.0
+Fetching data files...
+```
+
+#### References
+
+- [HTCondor Quick Start Guide](https://htcondor.readthedocs.io/en/latest/users-manual/quick-start-guide.html)
+- [HTCondor: condor_submit](https://htcondor.readthedocs.io/en/latest/man-pages/condor_submit.html)
+- [HTCondor: condor_transfer_data](https://htcondor.readthedocs.io/en/latest/man-pages/condor_transfer_data.html)
+
+### To an ARC-CE Computing Element
+
+Test job showing env where it run
+
+```xrsl
+&( executable = "/usr/bin/env" )
+( jobname = "arctest" )
+( stdout = "stdout" )
+( join = "yes" )
+( gmlog = "gmlog" )
+```
+
+```shell
+# Generating a proxy for ARC
+$ arcproxy --voms dteam
+Enter pass phrase for private key:
+Your identity: /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Jane Doe
+Contacting VOMS server (named dteam): voms2.hellasgrid.gr on port: 15004
+Proxy generation succeeded
+Your proxy is valid until: 2022-10-27 02:23:52
+```
+
+```shell
+# Sending a job using ARC
+# Select a CE
+$ lcg-info --list-ce --vo dteam --bdii ldap://lcg-bdii.egi.eu:2170 \
+    --attrs CEImpl --query 'CEImpl=*ARC-CE'
+# Example: arc01.grid.cyfronet.pl:2811/nordugrid-SLURM-grid-dteam
+# Get info about the selected CE
+$ ldapsearch -x -LLL -H ldap://lcg-bdii.egi.eu:2170
+    -b "Mds-Vo-Name=local,o=grid" \
+    '(&(objectClass=GlueCE)(GlueCEUniqueID=alex4.nipne.ro*))'
+# Submitting the job, the JOB_ID will be written on the output
+$ arcsub --jobdescrfile testjob.xrsl --computing-element alex4.nipne.ro
+Job submitted with jobid: gsiftp://alex4.nipne.ro:2811/jobs/....
+# Export JOB_ID to be used for other commands
+$ JOB_ID="gsiftp://alex4.nipne.ro:2811/jobs/...."
+# Monitoring the status of the job
+# Jobs will be in state Finished once completed
+$ arcstat "$JOB_ID"
+
+# Use -l parameter with arcstat to get more information on the status of the Job
+$ arcstat -l "$JOB_ID"
+
+# Retrieve output of finished job
+# Retrieve output files of the finished job, removing them from the server
+$ arcget "$JOB_ID"
+
+Results stored at: Ow7KmRKv71nuvw3Vp3UrRNqABFKDmABFKDmfJKDmABFKDmoeT5zn
+Jobs processed: 1, successfully retrieved: 1, successfully cleaned: 1
+```
+
+Instead of manually selecting a site, it's possible to do some automatic
+selection from CE registered in a central registry, such as `nordugrid.org`.
+
+```shell
+# Automatic selection on the CE in the nordugrid.org registry
+$ arcsub --jobdescrfile testjob.xrsl --registry nordugrid.org
+(...)
+gsiftp://vm3.tier2.hep.manchester.ac.uk:2811/jobs/...
+JOB_ID="gsiftp://vm3.tier2.hep.manchester.ac.uk:2811/jobs/.."
+
+# Use -l paramter to get more information on the status of the Job
+$ arcstat -l "$JOB_ID"
+```
+
+#### References
+
+- [ARC: submit a job](http://www.nordugrid.org/arc/arc6/users/submit_job.html)
+- [ARC client tools](http://www.nordugrid.org/arc/arc6/users/client_tools.html)
+
+### To a CREAM Computing Element
```

Comment:
so we need this given that it's not supported anymore?

Reply:
I will remove it if I can't get it to work, but the fact is that there are still production CREAM CE in the infrastructure, expected to be maintained locally.

Reply:
I've rephrased this part, I think that now it can probably kept like this.